content/en/blog/_posts/2022-12-06-kubernetes-1.26-blog.md

```diff
@@ -0,0 +1,210 @@
+
+### CRI v1alpha2 removed
+
+With the adoption of the [Container Runtime Interface](/docs/concepts/architecture/cri/) (CRI) and the [removal of dockershim](/blog/2022/02/17/dockershim-faq/) in v1.24, the CRI is the only supported and documented way through which Kubernetes interacts with different container runtimes. Each kubelet negotiates which version of CRI to use with the container runtime on that node.
+
+In the previous release, the Kubernetes project recommended using CRI version `v1`, but kubelet could still negotiate the use of CRI `v1alpha2`, which was deprecated.
+
+Kubernetes v1.26 drops support for CRI `v1alpha2`.
+That [removal](https://github.com/kubernetes/kubernetes/pull/110618) will result in the kubelet not registering the node if the container runtime doesn't support CRI `v1`. This means that containerd minor version 1.5 and older are not supported in Kubernetes 1.26; if you use containerd, you will need to upgrade to containerd version 1.6.0 or later **before** you upgrade that node to Kubernetes v1.26. This applies equally to any other container runtimes that only support the `v1alpha2`: if that affects you, you should contact the container runtime vendor for advice or check their website for additional instructions in how to move forward.
+
+### Progress on the CSI migration process
+
+Following the GA of the [core Container Storage Interface (CSI) Migration](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/625-csi-migration) feature in the previous release, CSI migration is an on-going effort that we've been working on for a few releases now, and this release continues to add (and remove) features aligned with the migration's goals.
+
+#### _Delegate FSGroup to CSI Driver_ graduated to stable
+
+This feature allows Kubernetes to [supply the pod's `fsGroup` to the CSI driver when a volume is mounted](https://github.com/kubernetes/enhancements/issues/2317) so that the driver can utilize mount options to control volume permissions.
+Previously, the kubelet would always apply the `fsGroup`ownership and permission change to files in the volume according to the policy specified  in the Pod's `.spec.securityContext.fsGroupChangePolicy` field.
+Starting with this release, CSI drivers have the option to apply the `fsGroup` settings during attach or mount time of the volumes.
+
+This feature allows Kubernetes to [supply the pod's `FSGroup` to the CSI driver when a volume is mounted](https://github.com/kubernetes/enhancements/issues/2317) so that the driver can utilize mount options to control volume permissions. Previously, the kubelet would always apply the `fsGroup`ownership and permission change to files in the volume according to the policy specified  in the `Pod.spec.securityContext.fsGroupChangePolicy` field. Starting with this release, CSI drivers have the option to apply the `fsGroup` settings during attach or mount time of the volumes.
```

Comment:
@xing-yang looks like line 53 to 55 got repeated from 57.

Reply:
Indeed, fixing it.